lib/shoulda/matchers/active_model/validate_inclusion_of_matcher.rb

```diff
@@ -302,12 +302,12 @@ def in_range(range)
           self
         end
 
-        def allow_blank(allow_blank = true)
+        def allow_blank
           @options[:allow_blank] = allow_blank
```

Comment:
You are right, I updated the code.

As a note we need to add specs to `allow_blank` and `allow_nil` qualifiers of this matcher.